app/upload/models.py

```diff
@@ -1,12 +1,39 @@
+from tkinter import CASCADE
 from django.db import models
 from hello_django.storage_backends import S3CertBucket
 from django.conf import settings
 
 # Create your models here.
 
+
 class CertUploadModel(models.Model):
     uploaded_at = models.DateTimeField(auto_now_add=True)
     if settings.USE_S3:
         file = models.FileField(storage=S3CertBucket())
     else:
         file = models.FileField(upload_to='')
+
+# One that gets meta data for a miner from a cert table
+
+
+class KnowYourMiner(models.Model):
```

Comment:
aahhhh so there was a problem with how we're running our initial migrations, which we didn't catch because @thea-exe and I already had the appropriate migrations built on our local filesystems. Fixed in next commit